municipal_finance/templates/explore.html

```diff
@@ -0,0 +1,55 @@
+{% extends "layout.html" %}
+{% load pipeline %}
+
+{% block title %}babbage.ui{% endblock %}
```

Comment:
we could do a better job here, this is very likely to slip through the cracks if we miss it :)